app/models/occupation_standard.rb

```diff
@@ -34,6 +36,22 @@ def clone_as_unregistered!(creator_id:, organization_id:)
     end
   end
 
+  def organization_title
+    organization.title
+  end
+
+  def occupation_title
+    occupation.title
+  end
+
+  def industry_title
+    industry.title
+  end
+
+  def creator_name
+    creator.name
+  end
```

Comment:
Rails question for you... do you have a rule of thumb you use for when you do this vs:
`delegate :title, to: :organization, prefix:true` or something similar for each of these? Is there a benefit -- beyond easier to search/find in the project -- to being more explicit like this?

Reply:
Oh!  How did I not know about the `prefix` option on `delegate`?  I was just trying to avoid Law of Demeter issues, which I am often not good about but then it sometimes comes back to bite me later.  I will implement `delegate` with `prefix`, that's much nicer, thanks!